src/vuex/modules/common/crudlDocument/store.js

```diff
@@ -24,6 +21,16 @@ export const mutations = {
     state.documents = result.documents
     state.total = result.total
   },
+  [TOGGLE_SELECT_DOCUMENT] (state, id) {
```

Comment:
I know but since the documents are fetched on the ready action of the component it is not a problem for them to be in the store (no global problem caused). If this is a problem for you then we should really discuss about when to put things in the vuex store.